torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -725,6 +725,11 @@ def _separate_input_attributes_from_arguments(
                 if fill_defaults:
                     onnx_attributes[param.name] = param.default
 
+        # pick up extra kwargs
+        extra_kwargs = set(kwargs).difference({param.name for param in param_schemas})
```

Comment:
So extra kwargs become part of onnx_attributes.

Could there be values that should become onnx_inputs?

But I guess either way this is not matching. 

What happens for `div.Tensor_mode`? How is it not export error when it has additional kwarg `rounding_mode` that is not even part of torchlib op signature?

Reply:
I am trying another way around. But long story short. "rounding mode" will be ignored if it's dispatched to aten_div. If you look ino `eval_function`, you will find in param_manipulation, we currently allow extra_kwargs, but they will be ignored after param_schema. I expect that we will have Ci breaks in onnx-script tests if we turn it on right now, as we don't filter useless attributes in onnx-script.

I will make sure extra attributes raises UnsupportedError in exporter side if and only if those attributes are not filtered ahead.